### PR TITLE
Meeting Minutes 2022-09-15

### PR DESCRIPTION
**Pull request by sthagen**
- [x] Meeting minutes draft before meeting
- [x] Transcription of meeting business
- [x] Ready for review and merge

### CODE REVIEW

File: meeting_minutes/220915_SARIF_TC_66.md [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+### 1.7.3 Members losing voting rights if they have not joined this meeting by the time it ends
+
+* Paul Anderson will lose voting status if he does not join before the end of this meeting
+### 1.7.4 Members who previously lost voting rights who are attending this meeting
+
+* Chris Wysopal will regain voting rights after the next meeting in case he joins it
+
+### 1.7.5 Members who have declared a leave of absence
+
+* Jeff Williams has declared a leave of absence until the first meeting in November
```
Comment (dmk42) [inline meeting_minutes/220915_SARIF_TC_66.md:72]: This error was mine.  Jeff's leave of absence is just for the month of October; he plans to be at the remaining September meeting.

**Review by dmk42 — CHANGES_REQUESTED** [review]
Looks good.  Just one change needed, which was my fault.

File: meeting_minutes/220915_SARIF_TC_66.md [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+
+* ACTION on David to keep pushing to close on the wikipedia page
+  * David kindly asks if someone can support
+  * ONGOING 
+* ACTION on Mary to continue to develop fixes scenario
+  * ONGOING 
+* ACTION on Michael to rename from master to main (URL)
+  * COMPLETE 
+* ACTION on Thanassis to present runtime-specific data (on September 15th)
+  * COMPLETE after this meeting's demo
```
Comment (katrinaoneil) [inline meeting_minutes/220915_SARIF_TC_66.md:48]: this was postponed, right?

File: meeting_minutes/220915_SARIF_TC_66.md [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+  * Software Bill of Materials (SBOM like CycloneDX, SPDX, and SWID)
+* Stefan: List reads good, best to analyze early what we can include and where to only offer least intrusive extension points
+* Michael:
+  * Agrees to not disrupt the existing format too much without clear benefits for existing and new users of SARIF
+  * What does fit in our "property bag" approach
+* Jeff:
+  * Updates to SARIF provides currently good list of problems
+  * SBOM approaches organize around components providing different perspectives with massive referencing
+    * Without balancing the impact i.e. not making a more comprehensible information item
+      * Interpretatbility degrades when unweighted external references grow in numbers
```
Comment (katrinaoneil) [inline meeting_minutes/220915_SARIF_TC_66.md:135]: Interpretatbility -> Interpretability

File: meeting_minutes/220915_SARIF_TC_66.md [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+  * Suggests a more balanced appraoch that adds evidence only when increasing the comprehensibility
```
Comment (katrinaoneil) [inline meeting_minutes/220915_SARIF_TC_66.md:136]: appraoch -> approach

File: meeting_minutes/220915_SARIF_TC_66.md [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+* Stefan: 
+  * Not convinced if SBOMs or Security Advisories will contain SARIF data but instead wil point to public reports (CVEs) as SARIF is more for intermediate states.
```
Comment (katrinaoneil) [inline meeting_minutes/220915_SARIF_TC_66.md:138]: wil-> will

File: meeting_minutes/220915_SARIF_TC_66.md (hunk 2 above)
Comment (dmk42) [reply meeting_minutes/220915_SARIF_TC_66.md:48]: That's right.

File: meeting_minutes/220915_SARIF_TC_66.md [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+* Aditya: Yes, comprehensibility is a frequent request from the code scanning customers
+* David: Suggests to continue this interesting discussion next time
+
+## 4.5 Roadmap review
+
+* Defered to next time
+
+## 4.6 Review outcomes of subgroup discussions
+
+* Defered to next time
```
Comment (katrinaoneil) [inline meeting_minutes/220915_SARIF_TC_66.md:160]: Defered -> Deferred

**Review by katrinaoneil — APPROVED** [review]
a couple of minor typos, but otherwise looks good!

File: meeting_minutes/220915_SARIF_TC_66.md (hunk 1 above)
Comment (dmk42) [reply meeting_minutes/220915_SARIF_TC_66.md]: The correction helps, but his leave of absence will not take effect until after the next meeting.  Someone reading the minutes in a vacuum might think he will not be allowed to attend the remaining September meeting.  How about "Jeff Williams intends to declare a leave of absence for the month of October."?

Comment (sthagen) [reply meeting_minutes/220915_SARIF_TC_66.md]: Thanks. Noted intentions

**Review by dmk42 — APPROVED** [review]
Thanks!

**Pull request merged**